common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -299,6 +309,42 @@ public Model getModelInfo() {
     return modelInfoConverter.apply(pipeline);
   }
 
+  /**
+   * Method to load mojo pipelines for shapley scoring based on configuration
+   *
+   * <p>Order of operations to preserve backwards compatibility:
+   * 1. if property or env var shapley.types.enabled is set, load pipelines based on that
+   * 2. if shapley.enabled is true load all pipelines
+   *
+   */
+  private void loadMojoPipelinesForShapley() {
+    if (ShapleyLoadOption.NONE.equals(ENABLED_SHAPLEY_TYPES)) {
```

Comment:
From `Effective Java` by Joshua Bloch.

>  Item 1: Consider static factory methods instead of constructors
[...] it allows an immutable class to make the guarantee that no two equal instances exist: a.equals(b) if and only if a==b. If a class makes this guarantee, then its clients can use the == operator instead of the equals(Object) method, which may result in improved performance. Enum types provide this guarantee.

Improved performance, null safe, and compile type checking.

`enum1 == "Nick"` -- compiler complains

vs

`enum1.equals("Nick")` -- uh oh, compiles